test/java/seedu/addressbook/parser/ParserTest.java

```diff
@@ -46,7 +46,7 @@ public void helpCommand_parsedCorrectly() {
         final String input = "help";
         parseAndAssertCommandType(input, HelpCommand.class);
     }
-    
+
```

Comment:
Be careful of unnecessary or accidental changes here as it can cause unnecessary merge conflict.